diff --git a/web/modules/contrib/field_permissions/field_permissions.module b/web/modules/contrib/field_permissions/field_permissions.module
--- a/web/modules/contrib/field_permissions/field_permissions.module	(revision 3c61200606f406f0e54ef53c9f5bfa4991d188c6)
+++ b/web/modules/contrib/field_permissions/field_permissions.module	(date 1731977151589)
@@ -125,7 +125,7 @@

   $user_role_storage = \Drupal::service('entity_type.manager')->getStorage('user_role');
   // Allow each plugin to add to or alter the form.
-  foreach ($plugins as $plugin) {
+  foreach ($plugins as $id => $plugin) {
     if ($plugin instanceof AdminFormSettingsInterface) {
       // Allow plugin to add to the form.
       $form['field_permissions'][$id] = [
